frontend/app/routes/_public+/intake+/$id+/dental-insurance.tsx

```diff
@@ -67,12 +89,17 @@ export default function AccessQuestion() {
         <div className="my-6">
           {options.length > 0 && (
             <InputRadios
-              id="born-type"
-              name="bornType"
+              id="dental-insurance"
+              name="dentalInsurance"
               legend={t('dental-insurance-question:legend')}
               options={options.map((option) => ({
-                children: option,
-                value: option,
+                children: (
+                  <div>
+                    <strong>{getNameByLanguage(i18n.language, option)}</strong>
+                    {`, ${option.id === 'yes' ? t('dental-insurance-question:option1') : t('dental-insurance-question:option2')}`}
```

Comment:
It might be worth making the `yes` id configurable (in a future PR?).

This works now because we have control over the mocks, but eventually the IDs may change depending on the (Power Platform) environment.

Reply:
Thanks for pointing it out! I'm leaving it like this for now since we don't know what it will look like on Power Platform, will come back to this in the future if refactoring needed.